src/Symfony/Bridge/Twig/Form/TwigRenderer.php

```diff
@@ -17,25 +17,14 @@
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
-class TwigRenderer extends FormRenderer implements TwigRendererInterface
+class TwigRenderer extends FormRenderer
 {
-    /**
-     * @var TwigRendererEngineInterface
-     */
     private $engine;
 
-    public function __construct(TwigRendererEngineInterface $engine, CsrfTokenManagerInterface $csrfTokenManager = null)
+    public function __construct(TwigRendererEngine $engine, CsrfTokenManagerInterface $csrfTokenManager = null)
```

Comment:
Not sure, shouldn't the Twig extension be sure that the underlying renderer engine supports Twig? By the way, if we did that change, that would mean that we could entirely drop the `TwigRenderer` class and use the parent `AbstractRendererEngine` class instead.